pipelines/nextflow/modules/genome_metadata/amend_genome_data.nf

```diff
@@ -26,8 +26,9 @@ process AMEND_GENOME_DATA {
     
     shell:
         '''
-        amend_genomic_data --genome_infile !{genome_json} --genome_outfile genome.json \
-            --INSDC_RefSeq_report_infile !{asm_report} --genbank_infile !{genbank_gbff} \
-            --brc4_mode !{params.brc_mode}
+        amend_genomic_data --genome_infile !{genome_json} \
+            --report_file !{asm_report} \
+            --genbank_file !{genbank_gbff} \
```

Comment:
We are still using` genbank_infile` in some python scripts, maybe we should use only a single parameter

Reply:
I must confess I would have changed several argument names, but I felt it was out of the scope of the PR (although I may have done a few due to readability or easy typos). Do you want me to change this one back to `genbank_infile`?